airbyte-webapp/src/packages/cloud/views/layout/SideBar/SideBar.tsx

```diff
@@ -101,6 +81,13 @@ const SideBar: React.FC = () => {
   const cloudWorkspace = useGetCloudWorkspace(workspace.workspaceId);
   const { show } = useIntercom();
   const handleChatUs = () => show();
+  const { location } = useRouter();
+
+  const menuItemStyle = (isActive: boolean) => {
```

Comment:
Is this the same implementation as the one in the non-cloud SideBar ? Could this be a utilty function that can be shared across the two? It doesn't depend on on the props or state of the component so it can stay outside of it.

Reply:
Most of the sidebar could be a utility function that is shared between the two 😅  but yes.